### DISCUSSION TITLE
DAI 2023-2024 - My Docker Compose application - Valentin Ricard

### DISCUSSION THREAD

**Discussion by knownasred** [heig-vd-dai-course/heig-vd-dai-course#112] [Show and tell]
[Link to Repo](https://github.com/valentin-ricard/dai-start-docker)

![image](https://github.com/heig-vd-dai-course/heig-vd-dai-course/assets/91870700/3dc3c6cc-6f5d-48f0-bb1c-814e84021480)

**Comment by ludelafo** [comment]
Everything looks good to me, however, I would still add the `data` directory to the gitignore as well for privacy reasons. I wouldn't share all my gigabytes of files on Git as well.. Do you see what I mean?
